Validator/IsDate.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Validator
+{
+    public partial class Validator
+    {
+        public static bool IsDate(string input)
+        {
+            if (String.IsNullOrEmpty(input))
+            {
+                return false;
+            }
+            try
+            {
+                DateTime.Parse(input);
```

Comment:
`DateTime` has a `TryParse` method. Maybe that would be more suitable? See what I did with [`IsFloat`](https://github.com/ByteBlast/Validator/blob/master/Validator/Validator.cs#L36).